Scripts/Runtime/Entities/TaskSystem/Job/Wrapper/CancelCompleteDataStreamAccessWrapper.cs

```diff
@@ -0,0 +1,27 @@
+using Anvil.Unity.DOTS.Jobs;
+using Unity.Jobs;
+
+namespace Anvil.Unity.DOTS.Entities.Tasks
+{
+    internal class CancelCompleteDataStreamAccessWrapper : AbstractAccessWrapper
+    {
+        public CancelCompleteDataStream CancelCompleteDataStream { get; }
+
+        public CancelCompleteDataStreamAccessWrapper(CancelCompleteDataStream cancelCompleteDataStream,
+                                                     AccessType accessType,
+                                                     AbstractJobConfig.Usage usage) : base(accessType, usage)
+        {
+            CancelCompleteDataStream = cancelCompleteDataStream;
+        }
+
+        public override JobHandle Acquire()
```

Comment:
Should we add the async suffix to this and `Release` so it's consistent with the rest of the lib?